pkg/reconciler/topology/partitionset/partitionset_reconcile.go

```diff
@@ -89,10 +89,11 @@ func (c *controller) reconcile(ctx context.Context, partitionSet *topologyv1alph
 	}
 
 	var matchLabelsMap map[string]map[string]string
+	dimensions := removeDuplicates(partitionSet.Spec.Dimensions)
```

Comment:
should we put it into validation/cel ?
that way we wouldn't get any duplicates, right ?

Reply:
@sttts wdyt?